README.md

```diff
@@ -70,6 +70,36 @@ By default, a Refinery process will register itself in Redis using its local hos
 In environments where domain name resolution is slow or unreliable, override the reliance on name lookups by specifying the name of the peering network interface with the `IdentifierInterfaceName` configuration option.
 See the [Refinery documentation](https://docs.honeycomb.io/manage-data-volume/refinery/) for more details on tuning a cluster.
 
+
+### Mixing Legacy and Environment & Services Rule Definitions
+
+With the change to support environemt and services in Honeycomb, some users will want to support both sending telemetry to a legacy dataset and a new environment called the same thing (eg `production`).
+
+This can be achomplihsed by leveraging the new `DataserPrefix` configuration property and then using that prefix in the rules definitions for the legacy datasets.
```

Comment:
```suggestion
This can be accomplished by leveraging the new `DatasetPrefix` configuration property and then using that prefix in the rules definitions for the legacy datasets.
```